src/app/faqs/faqs.component.html

```diff
@@ -0,0 +1,46 @@
+<!--
+ Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<div class="header">
+    <h1 class="title">How it Works</h1>
+    <p class="subtitle">
+        Answers to frequently asked questions
+    </p>
+    <img src="/Illustration.svg" />
+</div>
+
+<!-- Note: Wrapper div is required to prevent the side menu "sticky" layout
+     option from continuing beyond the bounds of the wrapper. -->
+<div class="content-wrapper">
+    <app-scrollable-side-menu class="side-menu" [sideMenuSections]="sideMenuSections">
+    </app-scrollable-side-menu>
+
+    <div class="content">
+        <div *ngFor="let faq of faqContent" class={{faq.selector}}>
+            <h2>{{faq.title}}</h2>
+
+            <mat-expansion-panel disabled expanded>
+                <div class="panel-content">
+                    <div class="text">
+                        <p [innerHTML]="faq.content">
```

Comment:
Why switching to innerHTML?

Reply:
because some of the contents have links to external resources. The only way to render the links as links would be to use `innerHTML`

Reply:
Oh.. cool.